docs/mobility/slam/_index.md

```diff
@@ -22,14 +22,20 @@ Breaking changes are likely to occur, and occur often.
 SLAM is an important area of ongoing research in robotics, particularly for mobile applications such as drones, boats, and rovers.
 
 The Viam SLAM service supports the integration of SLAM as a service on your robot.
-You can conduct SLAM with data collected live by a [RPlidar](/registry/examples/rplidar/) or with LIDAR data you provide in configuration, and easily view the map you build on the **SLAM library** tab of your location's page in the [Viam app](https://app.viam.com):
+You can conduct SLAM with data collected live by a [RPlidar](https://app.viam.com/module/viam/rplidar) or with LIDAR data you provide in configuration, and easily view the map you build on the **SLAM library** tab of your location's page in the [Viam app](https://app.viam.com):
 
 ![Completed SLAM maps in the SLAM library tab](/mobility/slam/view-map-page.png)
 
 ## Used With
 
 {{< cards >}}
-{{< relatedcard link="/build/configure/components/camera/" alt_title="RPlidar" required="yes" alt_link="/registry/examples/rplidar/">}}
+{{< relatedcard link="ml/vision/vision-odometry/"  required="yes" alt_link="https://app.viam.com/module/viam/rplidar">}}
+{{< relatedcard link="/build/configure/components/camera/" alt_title="RPlidar" required="yes" alt_link="https://app.viam.com/module/viam/rplidar">}}
+
+or
```

Comment:
Replace this with:
```suggestion
<div class="align-content-center">or</div>
```

And then add an item to the css file to do

```
.align-content-center {
    align-self: center
}
```

Reply:
Done-- I think right css file, lmk if not

Reply:
yep! all good!